plugins/updater/src/updater.rs

```diff
@@ -1026,6 +1050,40 @@ impl PathExt for PathBuf {
     }
 }
 
+#[cfg(windows)]
+fn escape_msi_property_args(args: &[&OsStr]) -> Vec<String> {
```

Comment:
I think this should operate on a single value, and return the escaped value and leave the loop stuff out at the calling site, it could just even be used with `iterator.map()`

Reply:
ok good idea